frontendFile/src/main/gen/com/virtuslab/gitmachete/frontend/file/grammar/MacheteEntry.java

```diff
@@ -0,0 +1,9 @@
+// This is a generated file. Not intended for manual editing.
```

Comment:
Hmm https://github.com/JetBrains/gradle-grammar-kit-plugin also looks like a viable (?) option that could make it possible to avoid checking-in the generated code... not sure if this limitation: _plugin does not support two-pass generation, therefore does not suport method mixin_ is sth that hurts us though?

Reply:
I'll take a look at that (on a separate PR)

Reply:
Done on #343